src/vs/workbench/electron-browser/commands.ts

```diff
@@ -423,3 +424,24 @@ export function registerCommands(): void {
 		});
 	});
 }
+
```

Comment:
@bpasero that makes this change smaller because `GlobalQuickOpenCommand.ID` is used here and there